runtimes/common/src/lib.rs

```diff
@@ -153,3 +158,35 @@ pub type FeeSplit<R, B1, B2> = SplitFeesByRatio<R, FeeSplitRatio, B1, B2>;
 /// https://w3f-research.readthedocs.io/en/latest/polkadot/Token%20Economics.html#-2.-slow-adjusting-mechanism
 pub type SlowAdjustingFeeUpdate<R> =
 	TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+pub struct Tippers<R, I>(PhantomData<R>, PhantomData<I>);
+impl<R, I: 'static> ContainsLengthBound for Tippers<R, I>
+where
+	R: pallet_membership::Config<I>,
+{
+	fn max_len() -> usize {
+		<R as pallet_membership::Config<I>>::MaxMembers::get() as usize
```

Comment:
Just to be on the safe side, perhaps use `saturated_into`.

Reply:
[1b469d9](https://github.com/KILTprotocol/mashnet-node/pull/352/commits/1b469d95681951ab772e747eb216d26d32fe8397)